salt/cloud/clouds/azurearm.py

```diff
@@ -10,6 +10,7 @@
 :depends:
     * `Microsoft Azure SDK for Python <https://pypi.python.org/pypi/azure>`_ >= 2.0rc5
     * `Microsoft Azure Storage SDK for Python <https://pypi.python.org/pypi/azure-storage>`_ >= 0.32
+    * `Microsoft Azure CLI <https://pypi.python.org/pypi/azure-cli>` >= 2.0.12
```

Comment:
IMHO, we could improve upon this by also returning a list of the requirements where `__virtual__` currently just returns `False`.

To do so, return a tuple like `(False, 'The following deps are required....')` instead

Reply:
Done.